homework-submission/react-hw/src/App.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import Description from "./components/Description";
+
+class App extends React.Component {
+  render() {
+    return (
+      <div>
+        <ul className="list">
+          <h1>ToDo Checklist</h1>
+          <li>
+            <Description todo={"Get out of bed"} deadline={"Wed Sep 13 2017"} />
```

Comment:
very nice passing to the props, you are in the future now so I won't tell that you could do it with an object instead 😅